samples/subsys/module_hello/sample.yaml

```diff
@@ -0,0 +1,12 @@
+sample:
+  description: Module Hello sample, application made entirely from
+    subsystems and drivers
+  name: module hello
+common:
+    tags: introduction
+    integration_platforms:
+      - native_posix
+tests:
+  sample.basic.hello-module:
+    tags: introduction
+
```

Comment:
This sample does not include any direct console output.  The hello_module subsystem produces only one log entry.  

- How do we configure this to automatically verify that *no* console output is produced?
- How do we configure this to automatically verify that specific log entries *are* produced?